packages/smooth_app/lib/pages/product/summary_card.dart

```diff
@@ -355,6 +351,39 @@ class _SummaryCardState extends State<SummaryCard> {
     );
   }
 
+  Widget _getHeader(ProductCompatibilityHelper helper) {
+    if (widget.isFullVersion) {
+      return Center(
+        child: Text(
```

Comment:
I guess you could compute the `Text` child once, and use it in both cases.

Reply:
Changes done.